examples/custom-vpc-with-vault/README.md

```diff
@@ -1,7 +1,7 @@
 # Custom VPC with HashiCorp Vault
 
-This example deploys a custom VPC with a bastion host behind an ELB (including a Route53 DNS record)
-and a Vault cluster behind an ELB (including a Route53 DNS record).
+This example deploys a custom VPC with a bastion host behind an ELB and a Vault cluster behind an
```

Comment:
Does ELB keep open the connection more than an hour?

Reply:
It's set to 60sec IDLE timeout and SSH did not auto-close the connection after >2min. So I guess its fine.